components/tabs/stories/tabs.stories.js

```diff
@@ -1,10 +1,24 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { version } from "../package.json";
 import { TabsGroups } from "./tabs.test.js";
-import { Template } from "./template.js";
+import { Template, QuietGroup, OverflowGroup, VerticalGroup, CompactGroup } from "./template.js";
 
 /**
  * Tabs organize content into multiple sections and allow users to navigate between them. The content under the set of tabs should be related and form a coherent unit.
+ * 
+ * ## Use icons consistently
+ * 
+ * Icons are optional, but don’t mix the use of icons in tabs if they are utilized. Navigation controls require a clear spacial relationship to one another, and mixing the use of icons can dramatically impact the visual balance and presence for each tab item.
+ * 
+ * ## Selected item
+ * Only one tab item can be selected at any given time. This property changes an individual tab item’s selected state. This is shown with a selection indicator (a gray-900 line) under or next to the selected tab item.
```

Comment:
Do you want this gray-900 with backticks around it?

Reply:
Oh good catch! That was a [copy/paste](https://spectrum.adobe.com/page/tabs/#Selected) I missed. Here's the fix: https://github.com/adobe/spectrum-css/pull/2925/commits/820272d9bf4bbc6ddc2916733436292d4268e9e5